### PR TITLE
data: alsonick

### PR DESCRIPTION
**Pull request by alsonick**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5121"></a>

### CODE REVIEW

File: alsonick.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Nicholas Njoki",
+  "type": "personal",
+  "bio": "I'm a full-stack developer using tools like TypeScript & React.",
+  "avatar": "https://notnick.io/pfp.png",
```
Comment (sidharthviz) [inline alsonick.json:5]: Hi @alsonick As per the recent updates, No need to add the avatar
You can remove it. Thank you

**Review by sidharthviz — CHANGES_REQUESTED** [review]

File: alsonick.json (hunk 1 above)
Comment (alsonick) [reply alsonick.json]: Thanks for the reply, I've now removed the avatar property. 🙂

**Review by sidharthviz — APPROVED** [review]
Looks Good 🎉

**Review by SurajPratap10 — DISMISSED** [review]
Thanks!

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**